WooCommerce/Classes/ViewRelated/JetpackSetup/JetpackSetupCoordinator.swift

```diff
@@ -273,6 +273,7 @@ private extension JetpackSetupCoordinator {
                         }
                     })
                 }
+                progressView.dismiss(animated: true)
```

Comment:
I suggest we move this to the `self.stores.synchronizeEntities` onCompletion, to make sure we are not dismissing the progress view until we finish all tasks, WDYT?

Reply:
Yes it makes sense, updated here 3b7238b9000e14f620e1e6f130fcadba242c358a